diff --git a/dist/index.js b/dist/index.js
index beef88fc6eaefe99415ce01cd22f9948e4a97f05..b63e534af6a094d5e06dee8f5a25208ba5b4fbe8 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3274,7 +3274,10 @@ function isParsableJson(input) {
 */
 function parseJsonEventStream({ stream, schema }) {
 	return stream.pipeThrough(new TextDecoderStream()).pipeThrough(new EventSourceParserStream$1()).pipeThrough(new TransformStream({ async transform({ data }, controller) {
-		if (data === "[DONE]") return;
+		if (data === "[DONE]") {
+			controller.terminate();
+			return;
+		}
 		controller.enqueue(await safeParseJSON({
 			text: data,
 			schema
